app/server/github.py

```diff
@@ -50,7 +50,7 @@ def push(self):
             if remotes[0].name != "origin":
                 repo.create_remote("origin", target_url)
             origin = repo.remote(name="origin")
-            origin.fetch()
+            origin.pull()
```

Comment:
```suggestion
            origin.fetch()
```

`.fetch()` now works.